localstack/utils/run.py

```diff
@@ -153,7 +153,7 @@ def _worker(*_args):
 
 def is_command_available(cmd: str) -> bool:
     try:
-        run("which %s" % cmd, print_error=False)
+        run(["which", cmd], print_error=False)
```

Comment:
minor: changing to the non-shell `subprocess` execution here, to prevent potential command injection.